.env-template

```
@@ -50,6 +50,12 @@ YAGNA_DATADIR="."
 #ZKSYNC_RINKEBY_RPC_ADDRESS=https://rinkeby-api.zksync.io/jsrpc
 #ZKSYNC_FAUCET_ADDR=http://3.249.139.167:5778/zk/donatex
 
+# ZkSync driver asserts additional funds available to make `N` transactions
+#ALLOCATE_FOR_N_TRANSACTIONS=10
```

`ALLOCATE_FOR_N_TRANSACTIONS` sounds kinda wayward to me. How about this?
```suggestion
#TRANSACTIONS_PER_ALLOCATION=10
```